Gopkg.lock

```diff
@@ -539,8 +539,8 @@
     "v2",
     "v2/fake"
   ]
-  revision = "39dbb4e91dfc6aad1f293e73fba18cc5f75eceb3"
-  version = "0.0.6"
+  revision = "c57d7cdbb99a641570f006b9a063c7376306a3e9"
+  version = "0.0.8"
```

Comment:
Did you intend to upgrade to 0.0.8 of the osb client? The manifest and your comment said "0.0.7". Just making sure that we are not relying on changes is 0.0.8 unintentionally.

Reply:
Ah, I didn't notice dep pulled in a later version. I'll change the dependency constraint to "=0.0.7".